Formula/vmdktool.rb

```diff
@@ -0,0 +1,21 @@
+class Vmdktool < Formula
+  desc "Converts raw filesystems to VMDK files and vice versa"
+  homepage "http://manned.org/vmdktool"
+  url "https://people.freebsd.org/~brian/vmdktool/vmdktool-1.4.tar.gz"
+  sha256 "981eb43d3db172144f2344886040424ef525e15c85f84023a7502b238aa7b89c"
+
+  def install
+    system "make", "CFLAGS='-D_GNU_SOURCE -g -O -pipe'"
+    bin.mkpath
+    man8.mkpath
+    # vmdktool makefile doesn't recognize --manpath option, so force it:
+    inreplace "Makefile", "man/man8", "share/man/man8"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    # Yes, it's a bad test, but vmdktool only does conversions of existing disk images
```

Comment:
Great suggestion! For my use cases I'm not generally creating new images, but this is a great way to validate this. Will do.